Multihost/basic-attestation/main.fmf

```diff
@@ -1,4 +1,12 @@
 summary: Tests basic keylime attestation scenario on multiple hosts
+description:
```

Comment:
Something like this should be the proper format of the description as it should be a text (https://tmt.readthedocs.io/en/stable/spec/tests.html#description)
```suggestion
description: |
```

Reply:
Thanks Dalibor, I tried to update changes as you suggested.